cmd/server/search_handlers.go

```diff
@@ -321,10 +326,23 @@ var (
 			resp.NonSDNMenuBasedSanctionsList = s.TopNS_MBS(limit, minMatch, name)
 		},
 	}
+
+	// eu - consolidated sanctions list
+	euGatherings = []searchGather{
+		func(s *searcher, _ filterRequest, limit int, minMatch float64, name string, resp *searchResponse) {
+			resp.EUCSL = s.TopEUCSL(limit, minMatch, name)
+		},
+	}
 )
 
+func generateAllGatherings() []searchGather {
+	gatherings = append(gatherings, cslGatherings...)
+	gatherings = append(gatherings, euGatherings...)
+	return gatherings
+}
```

Comment:
I don't think we want to append these to global state? With each request we'd grow the array of searchGathers over and over.

Reply:
good catch, I updated this to be a var block like before